apps/exploits/tests/test_api.py

```diff
@@ -60,6 +65,10 @@ def load_data():
     objects = process_data_epss(EPSS_TEST_FILE)
     store_objects(objects)
 
+    ps_module = PsModuleFactory(name="ps-module")
+    PsUpdateStreamFactory(active_to_ps_module=ps_module)
+    PsUpdateStreamFactory(active_to_ps_module=None)
```

Comment:
I may be wrong, but I do not think this combination testes the logic/query. It seems to me that you create one module with an update stream, and update stream without any associated module.

I think it should be more something like:
```python
    # This module is supported
    ps_module = PsModuleFactory(name="ps-module-1")
    PsUpdateStreamFactory(active_to_ps_module=ps_module)

    # This module is not supported, as there is no active update stream
    PsModuleFactory(name="ps-module-2")
```

Reply:
Updated.